dev-proxy-plugins/Inspection/DevToolsPlugin.cs

```diff
@@ -113,9 +118,37 @@ private string GetBrowserPath(DevToolsPluginConfiguration configuration)
 
     private Process[] GetBrowserProcesses(string browserPath)
     {
-        return Process.GetProcesses().Where(p =>
-            p.MainModule is not null && p.MainModule.FileName == browserPath
-        ).ToArray();
+        List<Process> processes = new List<Process>();
```

Comment:
I suggest we simplify it to something like:

```cs
    private Process[] GetBrowserProcesses(string browserPath)
    {
        return Process.GetProcesses().Where(p => {
            try
            {
                return p.MainModule is not null && p.MainModule.FileName == browserPath;
            }
            catch (Exception ex)
            {
                Logger.LogDebug("Error while checking process: {Ex}", ex.Message);
                return false;
            }
        }).ToArray();
    }
```

Since we're not doing anything with the caught exceptions, there's no point in keeping them around. For debugging, we can show the message inline.

Reply:
Yes, I agree, if you aren't interested in tracking the exception, your solution is cleaner and simpler 😊